README.md

```diff
@@ -50,15 +57,14 @@ As in the previous way, there are some optional parameters that you can specify
 The pagination strategy that **Fountain** is using can be seen in the following image:
 <br> <p align="center"> <img src="images/paginationStrategy.png" /> </p>
 
-The paging strategy starts with an initial service data request.
+It starts with an initial service data request.
 By default the initial data requested is three pages, but this value can be changed, in the [`PagedList.Config`](https://developer.android.com/reference/android/arch/paging/PagedList.Config.html), using the [`setInitialLoadSizeHint`](https://developer.android.com/reference/android/arch/paging/PagedList.Config.html#initialLoadSizeHint) method.
-This parameter can be set in the [`Fountain`](Listing.md) factory method. 
-When the service data comes, all data is refreshed in the [`DataSource`] using the [`CachedDataSourceAdapter`].
-Note that the [`Listing`](Listing.md) component will notify that the data changed.
+This parameter can be set in the factory constructor method. 
+When the service data comes from the service, all data is refreshed in the `DataSource` using the `CachedDataSourceAdapter`.
+Note that the `Listing` component will notify that data changed.
```

Comment:
maybe `will notify that the data has changed`?

Reply:
shouldn't be `will notify that data has changed`?